docker/Dockerfile.base

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:focal
+FROM ubuntu:jammy
```

Comment:
The officially supported platform for gz-sim9 is Ubuntu:noble. https://gazebosim.org/docs/latest/install/#supported-platforms